trivia.py

```diff
@@ -142,21 +142,21 @@ def signedOn(self):
         Actions to perform on signon to the server.
         '''
         self.join(self._game_channel)
-        self.msg('NickServ', 'identify %s' % config.IDENT_STRING)
-        print("Signed on as %s." % (self.nickname,))
+        self.msg("NickServ", "identify {}".format(config.IDENT_STRING))
+        print("Signed on as {}.".format(self.nickname))
```

Comment:
That comma is there I think just to make it a tuple, which looking at it isn't required I think.

It can be deleted.